app/views/jobs/interest.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class='container'>
+  <h1>List of the jobs sorted by level of interest: </h1>
+</div>
+<div class='container'>
+    <% @job_interest.sort.reverse.each do |key, value| %>
+      <h3>Level of Interest: <%= key %></h3>
+      <li><%= link_to value[0], company_job_path(Job.link(value[0]).company, Job.link(value[0])) %></li>
```

Comment:
same here - what part of MVC is responsible for grabbing our data before sending it to a view template?